lib/utils.h

```diff
@@ -78,4 +78,7 @@ inline static const char *getOrdinalText(int i)
     return "th";
 }
 
+template<typename T>
+inline void UNUSED(T&&) {}
```

Comment:
In my opinion, (void)param is not very readable and makes me think - what's the purpose of it.